.github/workflows/CI-MATs.yml

```
@@ -0,0 +1,105 @@
+      should_run: ${{ steps.check.outputs.should_run }}
+      build_environment: ${{ steps.get_env.outputs.environment }}
+    steps:
+      - id: check
+        run: |
+          if [[ "${{ github.event.workflow_run.conclusion }}" == "success" ]]; then
+            echo "should_run=true" >> $GITHUB_OUTPUT
+          else
+            echo "should_run=false" >> $GITHUB_OUTPUT
+          fi
+      - id: get_env
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:36: Double quote to prevent globbing and word splitting

(shellcheck)